api/controllers/CampaignBotController.js

```diff
@@ -241,6 +264,20 @@ class CampaignBotController {
    * @return {object}
    */
   loadCurrentSignup(req, id) {
+    this.debug(req, `loadCurrentSignup:${id}`);
+
+    if (this.isCommandClearCache(req)) {
+      this.debug(req, `isCommandClearCache`);
+
+      return app.locals.db.signups
+        .findByIdAndRemove(id)
```

Comment:
Uses https://github.com/DoSomething/gambit/pull/637/commits/c2e055111897ee53283962589072af8469619aec instead of deleting the Signup model.

If user texts in the clear cache command, it will reset their `user.campaigns` cache for all Campaigns (not just the current Campaign they're texting to).